brabbuss/refactor-tractor
# Accessibility first pass

### What does this PR do?
(*please link commit if helpful by copy/paste commit id*)
1. This PR is based on the first Lighthouse audit on our live site where we started with a 57%.

Key areas that I focused on were:
- Correctly labeling buttons with aria
- Minimum of an AA rating for contrast in each of our quadrants
- Testing useability in colorblindness cases with chrome extensions
- Replacing icons and removing extra png files.

### What are the relevant tickets (please link)?
1. #13 
2. #24 
3. #23 
4. #21 
5. #22 

### Where should the reviewer start?
1. index.html: line 2, and each **button** element(29 items), each **h2**(16 items) and **h3** elements(8 items).
2. styles.scss: lines #65-70, 130-136, each button styling element.
3. images: Added new svg icons for info, calendar, friends, trending, and goback.

### How should this be tested?
1. Feel free to start the server and launch the app. Open dev tools and run a Lighthouse audit. Use colorblindness filters and check contrast values.

### Any background context you want to provide?
1. Not at this time.

### Screenshots (if applicable)
<img width="1093" alt="Screen Shot 2020-10-26 at 3 43 51 PM" src="https://user-images.githubusercontent.com/21049650/97238483-2573e500-17af-11eb-8d68-1f04658976b8.png">

### New Questions/Thoughts?:
1. I believe there are more aria tags to be added for states of elements. With this application we want it to be very clear what area you are in.
2. I still need to actually run through the site myself with the screen reader.
